SimTracker/TrackAssociatorProducers/plugins/QuickTrackAssociatorByHitsImpl.h

```diff
@@ -5,6 +5,8 @@
 
 #include "SimDataFormats/Associations/interface/TrackToTrackingParticleAssociator.h"
 
+#include "SimTracker/TrackerHitAssociation/interface/ClusterTPAssociationProducer.h"
```

Comment:
OK, it isn't as bad here since it is still contained within the plugins directory, but on general principles I think what I said is still true, nothing should ever depend on a producer's code.